app/views/content/landing/how-to-fund-your-teacher-training/_content.html.erb

```diff
@@ -0,0 +1,109 @@
+    <h2 class="heading--box-blue">How can I fund my teacher training?</h2>
+
+    <div class="inset">
+      <p>
+        Different bursaries and scholarships are available depending on the subject you're training to teach.
+      </p>
+
+      <h3>Postgraduate bursaries and scholarships</h3>
+
+      <table class="first-column-headings right-aligned-values">
+        <caption>Postgraduate bursaries and scholarships are available for the subjects listed below.</caption>
+        <thead>
+          <tr>
+            <th scope="col">Subject</th>
+            <th scope="col">Bursary</th>
+            <th scope="col">Scholarship</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Biology</td>
+            <td>£20,000</td>
+            <td></td>
+          </tr>
+          <tr>
+            <td>Chemistry</td>
+            <td>£27,000</td>
+            <td>£29,000</td>
+          </tr>
+          <tr>
+            <td>Computing</td>
+            <td>£27,000</td>
+            <td>£29,000</td>
+          </tr>
+          <tr>
+            <td>Design and technology</td>
+            <td>£20,000</td>
+            <td></td>
+          </tr>
+          <tr>
+            <td>English</td>
+            <td>£15,000</td>
+            <td></td>
+          </tr>
+          <tr>
+            <td>Geography</td>
+            <td>£25,000</td>
+            <td></td>
+          </tr>
+          <tr>
+            <td>Languages</strong><br> (French, German and Spanish only)</td>
+            <td>£25,000</td>
+            <td>£27,000</td>
+          </tr>
+          <tr>
+            <td>Languages</strong><br> (all other languages, including ancient languages)</td>
+            <td>£25,000</td>
+            <td></td>
+          </tr>
+          <tr>
+            <td>Maths</td>
+            <td>£27,000</td>
+            <td>£29,000</td>
+          </tr>
+          <tr>
+            <td>Physics</td>
+            <td>£27,000</td>
+            <td>£29,000</td>
+          </tr>
+        </tbody>
+      </table>
+
+      <h3>Undergraduate bursaries</h3>
+
+      <p>You might be eligible for a bursary of £9,000 if you do:</p>
+
+      <ul>
+        <li>an undergraduate maths or physics course leading to qualified teacher status (QTS)</li>
+        <li>an opt-in undergraduate QTS course in computing, languages, maths or physics</li>
+      </ul>
+
+      <p><%= link_to("Find out more about how to get a bursary or scholarship", page_path("funding-and-support/scholarships-and-bursaries")) %>.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">More financial support</h2>
+
+    <div class="inset">
+      <p>
+        You can get a tuition fee and maintenance loan to fund your teacher training,
+        even if you've had student finance before.
+      </p>
+
+      <p>You could also be eligible for up to:</p>
+
+      <ul>
+        <li>£323.85 a week for 2 or more children if you're a parent</li>
```

Comment:
this feels a bit misleading @EllieNodder - it's not if you have children per se, it is for childcare costs if I'm reading it correctly?